hardhat.config.ts

```diff
@@ -263,7 +305,45 @@ const config: HardhatUserConfig = {
       arbitrumOne: process.env.ARBITRUM_API_KEY || "",
       optimisticGoerli: process.env.OPTIMISTIC_API_KEY || "",
       optimisticEthereum: process.env.OPTIMISTIC_API_KEY || "",
+      baseGoerli: process.env.BASE_API_KEY || "",
+      lineaGoerli: "PLACEHOLDER_STRING",
+      lineaMainnet: "PLACEHOLDER_STRING",
+      baseMainnet: process.env.BASE_API_KEY || "",
     },
+    customChains: [
+      {
+        network: "lineaGoerli",
```

Comment:
I think these wont work. I had to name it like "linea-goerli" here and above. cc @AmanRaj1608 
try some deployment and contract verification for this chains.

Reply:
seems to work:
https://explorer.goerli.linea.build/address/0x3FA4BE2a60F2Ac16f96DE47630784dD9FC9De77e/contracts#address-tabs